cli/pcluster/commands.py

```diff
@@ -92,88 +92,18 @@ def _upload_hit_resources(bucket_name, pcluster_config):
     bucket, key = match.group(1), match.group(2)
     file_contents = boto3.resource("s3").Object(bucket, key).get()["Body"].read().decode("utf-8")
 
-    hit_config = {
-        "cluster": {
-            "label": "default",
-            "default_queue": "multiple_spot",
-            "queue_settings": {
-                "multiple_spot": {
-                    "compute_type": "spot",
-                    "enable_efa": False,
-                    "disable_hyperthreading": True,
-                    "placement_group": None,
-                    "compute_resource_settings": {
-                        "multiple_spot_c4.xlarge": {
-                            "instance_type": "c4.xlarge",
-                            "min_count": 0,
-                            "max_count": 10,
-                            "spot_price": None,
-                            "vcpus": 2,
-                            "gpus": 0,
-                            "enable_efa": False,
-                        },
-                        "multiple_spot_c5.2xlarge": {
-                            "instance_type": "c5.2xlarge",
-                            "min_count": 1,
-                            "max_count": 5,
-                            "spot_price": 1.5,
-                            "vcpus": 4,
-                            "gpus": 0,
-                            "enable_efa": False,
-                        },
-                    },
-                },
-                "efa": {
-                    "compute_resource_settings": {
-                        "efa_c5n.18xlarge": {
-                            "instance_type": "c5n.18xlarge",
-                            "min_count": 0,
-                            "max_count": 5,
-                            "spot_price": None,
-                            "vcpus": 36,
-                            "gpus": 0,
-                            "enable_efa": True,
-                        },
-                    },
-                    "compute_type": "ondemand",
-                    "enable_efa": True,
-                    "disable_hyperthreading": True,
-                    "placement_group": "AUTO",
```

Comment:
placement_group is now set to DYNAMIC and not AUTO in the internal json config. We either need to transform DYNAMIC in AUTO or update the CloudFormation template.

You can test this by using a config with DYNAMIC placement_group and then submitting a slurm job to that partition:
```
sbatch --wrap "sleep 100" --partition queue
```

Then check no errors are present in `/var/log/parallelcluster/slurm_resume.log`. Currently you'd find `An error occurred (InvalidPlacementGroup.Unknown) when calling the RunInstances operation: The placement group 'DYNAMIC' is unknown` because DYNAMIC is being used as an id for an existing PG.

Reply:
As per our conversation, left to ``DYNAMIC`` and updated CloudFormation template and unit test.